docs/2-Interviews/questions_to_ask_the interviewer.md

```diff
@@ -1,6 +1,20 @@
 ## Questions to ask the interviewer
 
-At the end of an interview, you're often asked if you have any questions for *them*. Here are some of my favourite things to ask:
+At the end of an interview, you're often asked if you have any questions for *them*. It's important to ask some questions as a way of showing interest in the company, and identifying which company would be best suited for you.
+
+### Universally useful questions
+
+If you didn't have the time to prepare questions, if you already know everything because this is the 3rd interview with this company and you've already read through their whole website, fear not! Just ask them to talk about themselves. People love talking about themselves, you might not need to do any work at all! Plus, as they talk about themselves, they might say something interesting you can latch on to and converse about.
```

Comment:
> If you didn't have the time to prepare questions, if you already know everything because this is the 3rd interview with this company and you've already read through their whole website

I'm worried this will make it seem like those are the only cases to ask these questions - but they're definitely my go to no matter what